execution_environments/how_to_build_and_test_first_ee.rst

```diff
@@ -0,0 +1,181 @@
+.. _how_to_build_and_test_first_ee:
+
+******************************************************
+How To Build and Test Your First Execution Environment
+******************************************************
+
+* We assume that you installed all :ref:`required packages<how_to_prepare_environment>`.
+* For general information about Execution Environments, see the :ref:`Ansible Execution Environments Overview<ee_overview>` document.
+
+
+.. _build_first_ee:
+
+Building your first EE
+======================
+
+We are going to build an EE containing, in addition to standard packages like ansible-core and Python,
+an Ansible collection (``community.postgresql``) and its dependency (the ``psycopg2`` Python connector).
+
+The collection and the dependency can be replaced with any other package you need.
+
+1. Create and go to a project directory for your new EE:
+
+.. code-block:: bash
+
+  $ mkdir postgresql_ee && cd postgresql_ee
+
+2. Create the ``requirements.yml`` file which will specify collections you want to include in your EE.
+Tell it that you want the ``community.postgresql`` collection:
+
+.. code-block:: yaml
+
+  ---
+  collections:
+    - name: community.postgresql 
+
+3. Create the ``requirements.txt`` file which will list Python dependencies you want to include.
+In this how-to, you want the ``psycopg2-binary`` package:
+
+.. code-block:: bash
+
+  psycopg2-binary
+
+4. Create the ``execution-environment.yml`` definition file that specifies which content you want to include.
+List the files you created at previous steps:
+
+.. code-block:: yaml
+
+  ---
+  version: 1
+
+  dependencies:
+    galaxy: requirements.yml
+    python: requirements.txt
```

Comment:
We have Ansible Builder version 3, let us use it. 

```yaml
---
version: 3
images:
  base_image:
    name: quay.io/centos/centos:stream9

dependencies:
  ansible_core:
    package_pip: ansible-core==2.14.4
  ansible_runner:
    package_pip: ansible-runner
  galaxy: requirements.yml
  python: requirements.txt
```

For more information - https://ansible-builder.readthedocs.io/en/stable/definition/#version-3-format

Reply:
great, thanks for the link!